web/src/pages/Inbox/MessageDetail.js

```
@@ -0,0 +1,265 @@
+  const formatTimestamp = (timestamp) => {
+    const date = new Date(timestamp);
+    return date.toLocaleString();
+  };
+
+  const renderContent = (content, format) => {
+    if (!content) return null;
+
+    if (format === 'html') {
+      return (
+        <div 
+          dangerouslySetInnerHTML={{ __html: content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)